test/gtest/ucp/test_ucp_am.cc

```diff
@@ -900,11 +910,16 @@ UCS_TEST_P(test_ucp_am_nbx_single_multi_reply, zcopy, "ZCOPY_THRESH=1")
     test_am_send_recv(zcopy_fragment_size + 1, 0);
 }
 
-/* Tests check correctness of AM API when using UCP_AM_SEND_FLAG_REPLY flag.
- * Tests send messages with size equal to (fragment size + 1). The size should
- * be less than (fragment size + reply footer size) to check if the switch
- * between single and multi protocols is correct. */
-UCP_INSTANTIATE_TEST_CASE(test_ucp_am_nbx_single_multi_reply)
+UCS_TEST_P(test_ucp_am_nbx_reply_always, short_slow_path)
+{
+    /*
+     * This message is sent with UCP_OP_ATTR_FLAG_NO_IMM_CMPL so it will
+     * always go through AM short slowpath.
+     */
+    test_am_send_recv(8, 8, get_send_flag(), 0, UCP_OP_ATTR_FLAG_NO_IMM_CMPL);
```

Comment:
There is no need to call to `get_send_flag` here. `get_send_flag` is called inside `test_am_send_recv` when `send_am` is called.